opendatahub-io/opendatahub-operator
# Add workflow to sync dashboard crds automatically

Resolves: #200 
Prerequisite: https://github.com/opendatahub-io/odh-dashboard/pull/996

## Description
This PR awaits a dispatch from another workflow (https://github.com/maroroman/odh-dashboard/blob/239d23cd54addf29b88789d7a0b38f9a4dcc6313/.github/workflows/dispatch_on_crd_change.yaml) in the odh-dashboard repo and automatically creates a PR with these changes.

## How Has This Been Tested?
This PR was tested using https://github.com/nektos/act to locally verify that the workflow is sending the dispatch correctly.

To test this in real conditions, a fork of the operator and dashboard is required:

1. Create a pull request in the odh-dashboard fork that changes a file in /manifests/crd folder
2. Merge the PR into the main branch of the fork
3. Verify that the action has been triggered
4. Verify dispatch reached opendatahub-operator fork sync workflow and has triggered it.

## Merge criteria:
